core/src/test/java/org/opensearch/sql/executor/QueryServiceTest.java

```diff
@@ -134,7 +134,7 @@ Helper executeSuccess(Split split) {
           invocation -> {
             ResponseListener<ExecutionEngine.QueryResponse> listener = invocation.getArgument(2);
             listener.onResponse(
-                new ExecutionEngine.QueryResponse(schema, Collections.emptyList()));
+                new ExecutionEngine.QueryResponse(schema, Collections.emptyList(), ""));
```

Comment:
Do you need to pass in the empty string here? Doesn't the constructor on line 77 of `ExecutionEngine.java` cover this already?

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/217/commits/3de83d9c0a4932667b2078769d2a99c5167ea2fa